packages/nuxt-locokit/.prettierrc.yaml

```diff
@@ -0,0 +1,9 @@
+printWidth: 80
+semi: false
+useTabs: false
+editorconfig: true
+tabWidth: 2
+singleQuote: true
+trailingComma: 'all'
+bracketSpacing: true
+endOfLine: 'lf'
```

Comment:
We need to adopt the same rules through all our packages / workspaces.
Don't you think we have to put this file at the root level ? (same for others prettier files)

Reply:
These packages are necessary for eslint and prettier to work properly in designsystem. 

But just in case, we can check it #170